languages/translations/pt-BR.json

```diff
@@ -112,7 +112,7 @@
                 "upcoming": "Você vai chegar ao seu {nth} destino, em frente",
                 "short": "Você chegou",
                 "short-upcoming": "Você vai chegar",
-                "named": "Você chegou a (ao) {waypoint_name}, logo em frente"
+                "named": "Você chegou a {waypoint_name}, logo em frente"
```

Comment:
I backed out this change in favor of a grammar system that can automatically add a (mostly) appropriate article contraction: #294. Please have a look @jppcel @yuryleb.